src/Liquid/StandardFilters.php

```diff
@@ -201,6 +201,10 @@ public static function escape($input)
 			return $input;
 		}
 
+		if (is_null($input)) {
+			return $input;
```

Comment:
We have to cast it to string because the return type is string.

Reply:
Ok, I'll just `return '';` then.